OpenTrons/opentrons-api
# fix(app): fit tooltip width to content

# Overview

Previously, our app's `Tooltip` component with was defaulted tot 8.75rem, resulting in wide margins around short text. Here, I default the `maxWidth` style prop to 8.75rem instead, and default the `width` style prop to `max-content` to cut these margins off short text.

Closes RQA-2990

## Test Plan and Hands on Testing

- Start a protocol run on desktop
- Navigate back to setup tab and expand 'Deck hardware' tab
- Click disabled 'Proceed to labware position check' button and verify that tooltip width is fit to content
- Expand 'Labware position check' tab
- Click disabled 'Run Labware Position Check' button and verify that tooltip width maxes at 8.75 rem and wraps content
<img width="286" alt="Screenshot 2024-08-14 at 10 19 03 AM" src="https://github.com/user-attachments/assets/5ee7d9dc-8caf-4171-9202-3207ca81393f">

<img width="423" alt="Screenshot 2024-08-14 at 10 19 08 AM" src="https://github.com/user-attachments/assets/2fc9a686-5543-4fc0-b2b4-62d15e421fbb">

## Changelog

- update default `width` and `maxWidth` style props on `Tooltip`
- update test
- clean up imports on `SetupModuleAndDeck` while I was in there

## Review requests

see test plan

## Risk assessment

low. I looked through most other implementations of `Tooltip` to ensure I am not breaking style elsewhere
